core/controllers/resources.py

```diff
@@ -89,9 +89,14 @@ def get(self, page_context, page_identifier, asset_type, encoded_filename):
 
             # If the following is not cast to str, an error occurs in the wsgi
             # library because unicode gets used.
-            self.response.headers[
-                b'Content-Type'] = python_utils.convert_to_bytes(
-                    '%s/%s' % (asset_type, file_format))
+            if file_format == 'svg':
```

Comment:
Always try to minimize branching where possible. Define a new variable `content_type = '...' if file_format = 'svg' else ...` and then use that when setting the content type.